src/SFA.DAS.AssessorService.ApplyTypes/Roatp/GatewayReviewStatus.cs

```diff
@@ -7,5 +7,6 @@ public static class GatewayReviewStatus
         public const string InProgress = "In Progress";
         public const string Approved = "Approved";
         public const string Declined = "Declined";
+        public const string AskForClarification = "Ask Clarification";
```

Comment:
Should this be "Ask for Clarification" instead of "Ask Clarification"?

Reply:
I will change it to "Clarification" as "Ask For Clarification" goes over the limit (20) of the database field size. That was the reason for the strange name.
Please, look at Martin's suggestion.

Reply:
sounds good